epub33/rs/index.html

```diff
@@ -1591,22 +1618,23 @@ <h2>Vocabulary Association Mechanisms</h2>
 
 				<dt id="sec-property-datatype">The <code>property</code> Data Type</dt>
 				<dd>
-					<p>A Reading System MUST use the following rules to create an IRI [[RFC3987]] from a property:</p>
+					<p>A Reading System MUST use the following rules to create a URL [[URL]] from a property:</p>
 					<ul>
 						<li>
-							<p>If the property consists only of a reference, obtain the IRI by concatenating the IRI
-								stem associated with the <a href="https://www.w3.org/TR/epub-33/#sec-default-vocab"
-									>default vocabulary</a> [[EPUB-33]] to the reference.</p>
+							<p>If the property consists only of a reference, obtain the URL by concatenating the <a
+									href="https://url.spec.whatwg.org/#concept-base-url">base URL</a> [[URL]] associated
+								with the <a href="https://www.w3.org/TR/epub-33/#sec-default-vocab">default
+									vocabulary</a> [[EPUB-33]] to the reference.</p>
```

Comment:
Looking again, these probably shouldn't be references to the URL term. The vocabularies have "base URLs" defined for them, but because of the mashup of vocabularies in the authoring spec they're only base URLs in concept (i.e., each vocabulary doesn't actually have its own URL, only string values that reflect what they once were).

Reply:
I've rewritten the sections relating to property data types to not rely on the URL standard. These are effectively CURIEs and even when expanded they don't have to be parsed to URLs.

I've changed the RS section to "Expanding property Data Types" and tried to use this "expanding" phrasing along with "prefix URL" wherever these come up.

@iherman, you probably have the best grasp on whether these changes are in keeping with how curies are used, for example, in RDFa. Do these changes make sense to you?

Reply:
@mattgarrish looking at [RDFa CURIE Definition](https://www.w3.org/TR/rdfa-core/#s_curies) what you write looks correct to me.

However: the RDFa spec goes into great length specifying what happens if, say the prefix is an empty string, or the referent is empty, etc. Are we sure that those corner cases are covered? I mean we can say that a prefixes MUST be non-empty, and referents MUST be non-empty (it probably makes sense for our cases, which is much simpler than the vocabularies used in RDFa). Are these clearly specified somewhere?

Reply:
> the RDFa spec goes into great length specifying what happens if, say the prefix is an empty string

We're much more minimal. If a prefix URL isn't specified or it's not reserved then reading systems must ignore the value. That's why I found it odd that we didn't say anything if the resulting URL is invalid, and why we should correct that to match the rejection when a prefix doesn't have a URL.

> the referent is empty

That's another useful case to note. We should include another bullet to reject this case.